### PR TITLE
Gonzalo/langtonant

### PR DESCRIPTION
**Pull request by dulphyj**

### CODE REVIEW

File: bin/main/org/fundacionjala/at15/katas/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Katas folder
```
Comment (J0c4) [inline bin/main/org/fundacionjala/at15/katas/README.md:1]: Hi @dulphyj , please remove `bin/` folder (`git rm -r bin/`) and add it to `.gitignore`.

File: src/main/java/org/fundacionjala/at15/katas/langtonant/gonzalo/Grid.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    public Grid(int wide, int high){
+        this.wide=wide;
+        this.high=high;
+        grid=new char[wide][high];
+    }
+    
+    public void loadGrid(){       
+        for(int column=0; column<high;column++){
+            for(int file=0; file<wide;file++){
+                grid[column][file]='█';
```
Comment (J0c4) [inline src/main/java/org/fundacionjala/at15/katas/langtonant/gonzalo/Grid.java:16]: This char looks weird, which character is this?

File: src/main/java/org/fundacionjala/at15/katas/langtonant/gonzalo/Grid.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        }
+    }
+    
+    public int getPosition(int posX, int posY){
+        return grid[posX][posY];
+    }
+    
+      
+    public void changeValue(int posX, int posY){
+        if(grid[posX][posY]=='█'){
```
Comment (J0c4) [inline src/main/java/org/fundacionjala/at15/katas/langtonant/gonzalo/Grid.java:36]: It seems that there might be some encoding issues with your editor. Make sure you are using UTF-8

Comment (J0c4) [reply src/main/java/org/fundacionjala/at15/katas/langtonant/gonzalo/Grid.java:36]: If using VS Code, you can change encoding like this:
image

**Review by J0c4 — CHANGES_REQUESTED** [review]
Your code looks good! Few things pending:
- Please check my latests comments, they might help you with the errors currently thrown
- Fix checklist issues
- Fix merge conflicts

File: bin/main/org/fundacionjala/at15/katas/README.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+Welcome to katas folder for AT15!
```
Comment (J0c4) [inline bin/main/org/fundacionjala/at15/katas/README.md:2]: Please don't forget to remove `bin` folder

**Review by J0c4 — CHANGES_REQUESTED** [review]

File: bin/main/org/fundacionjala/at15/katas/README.md (hunk 4 above)
Comment (dulphyj) [reply bin/main/org/fundacionjala/at15/katas/README.md:2]: I did it

Comment (dulphyj) [reply bin/main/org/fundacionjala/at15/katas/README.md]: The bin folder was created again, delete it and push again, that should solve the problem

**Review by J0c4 — APPROVED** [review]

**Pull request merged**